src/patternfly/base/tokens/tokens-default.scss

```diff
@@ -203,56 +210,87 @@
   --pf-t--global--color--nonstatus--purple--100: var(--pf-t--color--purple--20);
   --pf-t--global--color--nonstatus--purple--200: var(--pf-t--color--purple--30);
   --pf-t--global--color--nonstatus--purple--300: var(--pf-t--color--purple--40);
+  --pf-t--global--color--nonstatus--purple--400: var(--pf-t--color--purple--50);
+  --pf-t--global--color--nonstatus--purple--500: var(--pf-t--color--purple--60);
   --pf-t--global--color--nonstatus--red--100: var(--pf-t--color--red--20);
   --pf-t--global--color--nonstatus--red--200: var(--pf-t--color--red--30);
   --pf-t--global--color--nonstatus--red--300: var(--pf-t--color--red--40);
+  --pf-t--global--color--nonstatus--red--400: var(--pf-t--color--red--60);
+  --pf-t--global--color--nonstatus--red--500: var(--pf-t--color--red--70);
   --pf-t--global--color--nonstatus--teal--100: var(--pf-t--color--teal--20);
   --pf-t--global--color--nonstatus--teal--200: var(--pf-t--color--teal--30);
   --pf-t--global--color--nonstatus--teal--300: var(--pf-t--color--teal--40);
+  --pf-t--global--color--nonstatus--teal--400: var(--pf-t--color--teal--70);
+  --pf-t--global--color--nonstatus--teal--500: var(--pf-t--color--teal--80);
   --pf-t--global--color--nonstatus--yellow--100: var(--pf-t--color--yellow--20);
   --pf-t--global--color--nonstatus--yellow--200: var(--pf-t--color--yellow--30);
   --pf-t--global--color--nonstatus--yellow--300: var(--pf-t--color--yellow--40);
+  --pf-t--global--color--nonstatus--yellow--400: var(--pf-t--color--yellow--70);
+  --pf-t--global--color--nonstatus--yellow--500: var(--pf-t--color--yellow--80);
   --pf-t--global--color--severity--critical--100: var(--pf-t--color--red-orange--60);
   --pf-t--global--color--severity--important--100: var(--pf-t--color--orange--50);
+  --pf-t--global--color--severity--important--200: var(--pf-t--color--orange--60);
   --pf-t--global--color--severity--minor--100: var(--pf-t--color--gray--50);
+  --pf-t--global--color--severity--minor--200: var(--pf-t--color--gray--80);
   --pf-t--global--color--severity--moderate--100: var(--pf-t--color--yellow--40);
+  --pf-t--global--color--severity--moderate--200: var(--pf-t--color--yellow--70);
   --pf-t--global--color--severity--none--100: var(--pf-t--color--blue--40);
+  --pf-t--global--color--severity--none--200: var(--pf-t--color--blue--60);
   --pf-t--global--color--severity--undefined--100: var(--pf-t--color--gray--30);
+  --pf-t--global--color--severity--undefined--200: var(--pf-t--color--gray--60);
   --pf-t--global--color--status--custom--100: var(--pf-t--color--teal--60);
   --pf-t--global--color--status--custom--200: var(--pf-t--color--teal--70);
+  --pf-t--global--color--status--custom--300: var(--pf-t--color--teal--80);
   --pf-t--global--color--status--danger--100: var(--pf-t--color--red-orange--60);
   --pf-t--global--color--status--danger--200: var(--pf-t--color--red-orange--70);
-  --pf-t--global--color--status--danger--300: var(--pf-t--color--red-orange--70);
```

Comment:
It doesn't look like we use this anywhere but technically this would be a breaking change, wouldn't it?

Reply:
Looks like that one was just moved in the file https://github.com/patternfly/patternfly/blob/45c709425978d5819d99ff1d8a99deba573b36e2/src/patternfly/base/tokens/tokens-default.scss#L57

Reply:
🤨 I thought we were sorting them...wonder if that broke.